accounts/tests/serializers/test_user_create.py

```diff
@@ -0,0 +1,45 @@
+from django.test.testcases import TestCase
+from accounts.serializers import UserCreateSerializer
+from unittest.mock import patch
+from firebase_admin import auth
+from rest_framework.serializers import ValidationError
+
+
+class UserCreateSerializerTestCase(TestCase):
+    def setUp(self):
+        self.data = {"email": "test@asd.com"}
+
+    @patch("firebase_admin.auth.create_user")
+    def test_user_create(self, mock_create):
+        uid = "test_uid"
+        serializer = UserCreateSerializer(data=self.data)
+        # anonymous object creation
+        firebase_user = type("", (object,), {"uid": uid})()
+        mock_create.return_value = firebase_user
+        self.assertTrue(serializer.is_valid())
+        user = serializer.save()
+        self.assertEqual(user.firebase_uid, uid)
+        mock_create.assert_called_once()
```

Comment:
nit: can we be extra granular and use `assert_called_once_with(args)` instead of `assert_called_once()`? (just to be extra careful with auth stuff)